examples/tfengine/generated/team/project_secrets/terraform.tfvars

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+project_id      = "example-prod-secrets"
+parent_type     = "folder"
+parent_id       = "12345678"
+billing_account = "000-000-000"
+apis            = ["secretmanager.googleapis.com"]
+
+
+
+shared_vpc_attachment = {
```

Comment:
Is this okay to have it say "\<no value>"? Seems like in that case this variable should be null.

Reply:
My bad, it was solved with the default you suggested [here](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/980#discussion_r682703322)